src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -229,45 +210,6 @@ public void fetchPetSaCredentials() {
     logger.debug("Enabled pet SA impersonation");
   }
 
```

Comment:
I thought you already deleted this in another PR.

Reply:
Gone after rebase